### DISCUSSION TITLE
Incorrect Timestamp When Sending Opus DTX (Discontinuous Transmission) Packets To Live Endpoint.

### DISCUSSION THREAD

**Discussion by tomiwu** [deepgram/community#977] [General help]
I’m encountering an issue while sending live audio to the WebSocket endpoint wss://api.deepgram.com/v1/listen. Specifically, the timestamp returned in the response (the `start` field) is incorrect when I send Opus DTX (discontinuous transmission) packets, which represent silent periods.

Could you clarify if this is expected behavior when sending DTX packets? If not, how can I address it?

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- The deepgram product you are using (e.g Speech to Text, Agent API)
- A request ID that triggered your error or issue.
